### PR TITLE
Added new opendkim counters.

### PR DESCRIPTION
**Pull request by knightorc**

### CODE REVIEW

File: postfix_stats.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```
Comment (knightorc) [inline postfix_stats.py:1]: Odd. I wonder if packaging the rpm changes this.

Comment (jharley) [inline postfix_stats.py:1]: Likely?  Could revert, but unlikely strictly necessary :)

**Comment by jharley** [comment]
:+1: LGTM :+1:

**Comment by jharley** [comment]
Left the branch intact in the event that you want to open an upstream PR, @knightorc

**Pull request merged**